analysis/combiners.py

```diff
@@ -644,6 +645,169 @@ def explain_computation(self):
         pass
 
 
+class SumAggregateErrorMetricsCombiner(pipeline_dp.Combiner):
+    """A combiner for aggregating errors across partitions for Sum"""
+    AccumulatorType = AggregateErrorMetricsAccumulator
+
+    def __init__(self, error_quantiles: List[float]):
+        # The contribution bounding error is negative, so quantiles <0.5 for the
+        # error distribution (which is the sum of the noise and the contribution
+        # bounding error) should be used to come up with the worst error
+        # quantiles.
+        self._error_quantiles = [(1 - q) for q in error_quantiles]
```

Comment:
Nit: this comment is present twice (here and in count combiner). WDYT about extracting this conversion to a helper function and having the comment there?
Another option is to have a common base class for both combiners